domains/noto/internal/noto/handler_mint.go

```diff
@@ -96,6 +105,17 @@ func (h *mintHandler) Assemble(ctx context.Context, tx *types.ParsedTransaction,
 		return nil, err
 	}
 
+	if !tx.DomainConfig.IsV0() {
```

Comment:
Is there a reason we don't include this for V0 Noto? Is there something about the new Solidity in V1 that enables manifests?

Reply:
It seems like this is a purely off-chain behavior and not tied to the ABI of Noto V0 vs V1.